WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1486,10 +1489,31 @@ def getSiteMask( self, siteState = 'Active' ):
     return S_OK( siteList )
 
 #############################################################################
-  def getSiteMaskStatus( self ):
+  def getSiteMaskStatus( self, sites = None ):
     """ Get the currently site mask status
     """
-    cmd = "SELECT Site,Status FROM SiteMask"
+    if isinstance(sites, list):
+
+      cmd = "SELECT Site, Status FROM SiteMask WHERE"
```

Comment:
Why not use
```... WHERE Site in ('site1', 'site2')```
That also works for one site and would simplify this function a lot.